File: Java/2823.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double count = 0;
        int n = scanner.nextInt();
        for(int i = 0; i < n; i++){
            double a = scanner.nextDouble(), b = scanner.nextDouble();
            count += a/b;
        }
        if(count <= 1){
            System.out.println("OK");
        }else{
            System.out.println("FAIL");
        }
    }
}

File: C/2823.c
#include <stdio.h>
 
int main() {
    float c = 0;
    int n; scanf("%d", &n);
    for(int i = 0; i < n; i++){
        double a, b; scanf("%lf %lf", &a, &b);
        c += a/b;
    }
    if(c <= 1)
        printf("OK\n");
    else
        printf("FAIL\n");
}
